.github/workflows/ci-actions.yml

```diff
@@ -110,7 +110,7 @@ jobs:
         java :
           - { name: Java8,
               java-version: 8,
-              maven_args: "-pl !integration-tests/gradle -pl !integration-tests/kubernetes/quarkus-standard-way"
+              maven_args: "-pl !integration-tests/gradle -pl !integration-tests/kubernetes/quarkus-standard-way -pl !integration-tests/kubernetes/maven"
```

Comment:
Hello @geoand, I don't see `integration-tests/kubernetest/maven` in our repo. Did you mean `integration-tests/maven` instead?

Reply:
Sorry, yeah :)

Reply:
I changed the PR around a bit